### DISCUSSION TITLE
How to recognize the digital numbers in the rectangle box with paddleocr?

### DISCUSSION THREAD

**Discussion by githubthunder** [PaddlePaddle/PaddleOCR#12447] [Q&A]
PaddleOCR fails to recognize the digital numbers in the rectangle box. can anyone help to solve that?
<img width="551" alt="sample" src="https://github.com/PaddlePaddle/PaddleOCR/assets/30004341/01a58d72-74d9-439d-836b-2bb494f375fb">

**Comment by GreatV** [comment]
<img width="551" alt="333808456-01a58d72-74d9-439d-836b-2bb494f375fb" src="https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/5a31bb1f-4531-477d-82e6-0ba3cb517ef1">

![result](https://github.com/PaddlePaddle/PaddleOCR/assets/17264618/6169470f-ef64-4f58-8ceb-1fa89a711606)



try lang=ch
